releasenotes/notes/node-reverse-traversal-dag-81dd7ac00bd7dcf0.yaml

```diff
@@ -0,0 +1,25 @@
+---
+features_transpiler:
+  - |
+    The :meth:.DAGCircuit.topological_op_nodes and :meth:.DAGCircuit.topological_nodes methods now support a reverse boolean argument. 
+    When set to True, the methods yield nodes in a reverse topological order, from the outputs of the circuit towards the inputs. 
+    This provides a direct and efficient way to iterate over a DAG backwards without the overhead of creating a new, structurally reversed 
+    DAG with :meth:.DAGCircuit.reverse_ops.
+
+    For example::
```

Comment:
Since the release notes list tend to get long, and because this is a simple change, I would drop the code example in this case.